mocks/handlers/insee/token.ts

```diff
@@ -0,0 +1,6 @@
+export const token = {
```

Comment:
why not just returning dummy rathen than actual payload ? As it will never be send to Insee API